### PR TITLE
Schedule pickups

### PR DESCRIPTION
**Pull request by Gbaja**

### CODE REVIEW

File: src/routes/routes.js [hunk 1]
```diff
@@ -49,6 +54,20 @@ router.post('/login_parent', login_parent.post);
 router.post('/register_parent', register_parent.post);
 router.post('/add_child', add_child.post);
 
+router.get('/schedule_pickup', (req, res) => {
+  parent_children_and_da(req.session.parent_id).then((queryRes) => {
```
Comment (emilyb7) [inline src/routes/routes.js:58]: yess, this is the proper way to handle a promise 👍 (related to my comments in #29 )

File: src/routes/schedule_pickup.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const parent_child_da_id = require('../queries/parent_child_da_id')
+
+exports.post = (req, res) => {
+//  console.log(req.body);
+  const schedule_details = req.body;
+  // console.log(parent_child_da_id(req.session.parent_id, schedule_details.child_name, schedule_details.da_name));
+   parent_child_da_id(req.session.parent_id, schedule_details.child_name, schedule_details.da_name).then((queryRes)=>{
+  //  console.log(queryRes);
+    return queryRes;
+  })
```
Comment (emilyb7) [inline src/routes/schedule_pickup.js:10]: please get rid of all the comments before merging :) and sort out your indentation!

File: src/validators.js [hunk 3]
```diff
@@ -11,14 +11,20 @@ const validateEmail = (email) => {
   })
 }
 
-const formatDate = (date) => {
-  const splitDate = date.split('-')
-  const reshuffle = splitDate[2] +"-"+ splitDate[1] + "-"+ splitDate[0];
-  return reshuffle;
+
+const make_names_array = (obj, which_names) => {
+  var arrayObj = obj.reduce(function(acc, el){
```
Comment (emilyb7) [inline src/validators.js:16]: just a naming thing: `obj` doesn't sound like something you should be able to reduce on. Maybe call it something better?

**Review by emilyb7 — COMMENTED** [review]
:)

**Review by sophielevens — APPROVED** [review]

**Review by Dragomirc — APPROVED** [review]

**Pull request merged**